CommunityToolkit.App.Shared/Renderers/ToolkitSampleRenderer.xaml

```diff
@@ -9,8 +9,10 @@
       xmlns:renderer="using:CommunityToolkit.App.Shared.Renderers"
       xmlns:wasm="http://uno.ui/wasm"
       xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:xamarin="http://uno.ui/xamarin"
```

Comment:
@Arlodotexe @michael-hawker I got this using from the `Uno.CommunityToolkit` sample app repo.. and it works, but not sure if this is the right one :-)?

Reply:
@Arlodotexe thoughts on this bit?

Reply:
Not sure if they'll be keeping `xamarin` long term, if we're keeping this we should use `not_win` instead.

Though I think this becomes extra if we create the composed (not inherited) markdown textblock, since we can use our C# compilation conditionals.